src/Socialengine/SnifferRules/Command/SniffCommand.php

```diff
@@ -232,4 +136,69 @@ protected function getOptions()
     {
         return array();
     }
+
+    protected function processOptions()
+    {
+        $standards = $this->config->get('sniffer-rules.standard', array('PSR2'));
+        $files = $this->config->get('sniffer-rules.files', array('app/models', 'app/controllers'));
```

Comment:
You mean like [a Laravel specific standard](https://github.com/antonioribeiro/laravelcs)?